styles/shinigami-eyes/catppuccin.user.css

```diff
@@ -0,0 +1,143 @@
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document
+  domain("facebook.com"),
+  domain("youtube.com"),
+  domain("reddit.com"),
+  domain("twitter.com"),
+  domain("x.com"),
+  domain("medium.com"),
+  domain("disqus.com"),
+  domain("tumblr.com"),
+  domain("wikipedia.org"),
+  domain("rationalwiki.org"),
+  domain("cohost.org"),
+  domain("bsky.app"),
+  domain("anarchism.space"),
+  domain("aus.social"),
+  domain("c.im"),
+  domain("chaos.social"),
+  domain("eightpoint.app"),
+  domain("eldritch.cafe"),
+  domain("fosstodon.org"),
+  domain("hachyderm.io"),
+  domain("infosec.exchange"),
+  domain("kolektiva.social"),
+  domain("mas.to"),
+  domain("masto.ai"),
+  domain("chaosfem.tw"),
+  domain("mastodon.art"),
+  domain("mastodon.cloud"),
+  domain("mastodon.green"),
+  domain("mastodon.ie"),
+  domain("mastodon.nz"),
+  domain("mastodon.online"),
+  domain("mastodon.scot"),
+  domain("mastodon.social"),
+  domain("mastodon.world"),
+  domain("mastodon.xyz"),
+  domain("mastodonapp.uk"),
+  domain("meow.social"),
+  domain("mstdn.ca"),
+  domain("mstdn.jp"),
+  domain("mstdn.social"),
+  domain("octodon.social"),
+  domain("ohai.social"),
+  domain("pixelfed.social"),
+  domain("queer.party"),
+  domain("sfba.social"),
+  domain("social.transsafety.network"),
+  domain("tech.lgbt"),
+  domain("techhub.social"),
+  domain("toot.cat"),
+  domain("toot.community"),
+  domain("toot.wales"),
+  domain("vulpine.club"),
+  domain("wandering.shop"),
+  domain("lgbtqia.space"),
+  domain("threads.net"),
+  domain("duckduckgo.com"),
+  domain("bing.com"),
+  domain("google.ar"),
+  domain("google.at"),
+  domain("google.be"),
+  domain("google.ca"),
+  domain("google.ch"),
+  domain("google.co.uk"),
+  domain("google.com"),
+  domain("google.de"),
+  domain("google.dk"),
+  domain("google.es"),
+  domain("google.fi"),
+  domain("google.fr"),
+  domain("google.is"),
+  domain("google.it"),
+  domain("google.no"),
+  domain("google.pt"),
+  domain("google.se")
+{
+
+  @media (prefers-color-scheme: light) {
+    :root {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    :root {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+
+    @accent-color: @catppuccin[@@lookup][@@accent];
```

Comment:
I don't think these should be changed from the template. We may decide to change the colors at some point, etc.

Reply:
I mean, yea, I can probably at like the "unused" color back?

Reply:
Huh? Just paste the full declarations from the template over what you have here.

Reply:
> Huh? Just paste the full declarations from the template over what you have here.

What you mean by "full declarations" may I ask?

Reply:
The variable declarations. https://github.com/catppuccin/userstyles/blob/9b1a1cfb30553b9cc5c98e5a4982793267fac101/template/catppuccin.user.css#L50-L76

Reply:
https://github.com/catppuccin/userstyles/pull/1026/commits/a09d19f075f5318e964477aa3de8f552fcebb1c5